docs/tutorial2023_unfolding/unfolding_exercise.md

```diff
@@ -0,0 +1,256 @@
+```
+
+> Note that it is important to add the option `--redefineSignalPOIs [list of parameters]`, to produce the impacts for all POIs we defined when the workspace was created with the `multiSignalModel`.
+
+After the initial fit is completed we can perform the likelihood scans for each nuisance parameter as shown below
+
+```shell
+combineTool.py -M Impacts -d ws_full.root -m 125 --robustFit 1 --doFits --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --job-mode condor --task-name impacts_zh --X-rtd FAST_VERTICAL_MORPH 
+```
+We will submit the jobs to the HTCondor to speed up the process.
+
+Now we can combine the results into the `.json` format and use it to produce the impact plots.
+
+```shell
+combineTool.py -M Impacts -d ws_full.root -m 125 --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --output impacts.json 
+
+plotImpacts.py -i impacts.json -o impacts_r_zh_75_150 --POI r_zh_75_150
+```
+
+![](figures/impacts_zh_75_150.png)
+* Do you observe differences in impacts plots for different POIs, do these differences make sense to you? 
+
+## Unfolded measurements
+
+Now that we studied the NP impacts for each POI, we can finally extract the measurements.
+Note, that in this exercise we are skipping further checks and validation that you should do on your analysis for the purposes of the tutorial. Namely the goodness of fit test and the post-fit plots of folded observables. Both of these checks were detailed in the previous exercises, you can find the description under the following links. 
+
+At this stage we'll run the `MultiDimFit` again scanning each POI to calculate the intervals, but this time we'll remove the `-t -1` option to extract the unblinded results. 
+
+Also since we want to unfold the measurements to the gen-level observables, i.e. extract the cross sections, we have to remove the theoretical uncertainties affecting the rates of signal processes, we can simply do this be freezing them `--freezeNuisanceGroups <group_name>`, using the `group_name` you assigned earlier in the tutorial. 
+
+Plot the scans and collect the measurements in the json file `summary_zh_stxs.json`. 
+
+```shell
+python scripts/plot1DScan.py scan_r_zh_75_150.root -o r_zh_75_150 --POI r_zh_75_150 --json summary_zh_stxs.json  
+``` 
+
+![](figures/scan_plot_r_zh_75_150.png)
+
+Repeat the same command for other POIs to fill the `summary_zh_stxs.json`, which can then be used to make the cross section plot by multiplying the standard model cross sections by the signal strengths' best-fit values as shown below. 
+
+```shell
+python scripts/make_XSplot.py summary_zh_stxs.json
+```
+![](figures/stxs_zh.png) 
+
+## POIs correlations
+
+In addition to the cross-section measurements it is very important to publish correlation matrix of measured cross sections.  
```

Comment:
"to publish the correlation matrix" (missing the) "of the measured cross sections" (again missing the)